### ISSUE TITLE
Add an option to change the brightness speed？Like the volume.

### ISSUE THREAD

**Issue by o000u** [irzyxa/Volume2#184]
Then,the brightness can also be set to 16 segments like volume adjustment, etc.
Thank you very much for developing such excellent software！！

**Comment by irzyxa** [comment]
You can configure this in current version:
![image](https://user-images.githubusercontent.com/5119360/150844208-00c0e8a3-61d7-4c16-9c3a-e1e8194dfc78.png)

**Comment by irzyxa** [comment]
![image](https://user-images.githubusercontent.com/5119360/150844887-6dace62d-0da1-4bc7-8b71-af653511eee8.png)

**Comment by irzyxa** [comment]
But you can change volume speed. I do not know where did you get 16 segments. Could you add some screenshots?

**Comment by o000u** [comment]
OK, unfortunately, I'm using IMAC now. Maybe I'll add some screenshots when I'm using PC tomorrow, okay? Thank you for your patience!^_^

**Comment by o000u** [comment]
Sorry, I can't find a way to add screenshots to the reply.
Anyway,my suggestion is that the brightness can also adjust the speed like the volume.

> But you can change volume speed. I do not know where did you get 16 segments. Could you add some screenshots?

**Comment by BenJewell** [comment]
Why are there random brightness options in this menu? The shortcuts don't even do anything for me. As far as I know this program only does audio and there's nothing else related to brightness. Right?

**Comment by irzyxa** [comment]
@BenJewell yes, Volume2 has function to control brightness. Please look at screenshots above how to configure it.

**Comment by irzyxa** [comment]
@o000u I will add this parameter to User Variables:
![image](https://user-images.githubusercontent.com/5119360/164013452-9c630986-e810-4e11-9496-a8e380fe0c07.png)
